src/styles/Theme.js

```diff
@@ -1,18 +1,18 @@
 import { ThemeProvider } from "styled-components";
 
-// example
 const theme = {
     colors: {
-      powderWhite: "#FFFDF9",
-      persianGreen: "#06B49A",
-      lightBlue: "#AFDBD2",
-      onyx: "#36313D"
+      darkGreen: "#064420",
+      lightGreen: "#e4efe7",
+      beige: "#faf1e6",
+      lightBeige: "#fdfaf6"
     },
```

Comment:
these should be named "primary" "secondary" "accent" etc because in light mode our dark green colours that we use for text may very well be a light green colour in dark mode (ie. the names will be inconsistent across different themes ya know)

Reply:
ah i was just going by the example given in the template, but yes that makes sense! 

@christylo i'm not sure what you had in mind in terms of how we're using each colour, but this is the colour definition that i ended up going with in the code - please let me know if my assumptions are wrong! 

![image](https://user-images.githubusercontent.com/62684843/121432868-1140ba80-c930-11eb-9e32-5919f64fe7e1.png)